frontends/vuejs/Dockerfile

```diff
@@ -7,15 +7,11 @@ COPY . .
 FROM base as development
 CMD npm run serve
 
-FROM base as build-stage
-RUN npm run build
-
-FROM nginx:stable-alpine as production
-RUN apk -U --no-cache add bash
+FROM base as production
+RUN apk -U --no-cache add bash nginx
+RUN mkdir /run/nginx
 COPY support/nginx.conf /etc/nginx/conf.d/default.conf
-COPY support/apply-env-vars.sh /
-
-COPY --from=build-stage /app/dist /app
+#COPY support/apply-env-vars.sh /
 
 EXPOSE 80
-CMD bin/bash -c "/apply-env-vars.sh && exec nginx -g 'daemon off;'"
```

Comment:
The reason this was working is that the `WORKDIR` was `/` ... so `bin/bash` was the same as `/bin/bash` 

if there had been a `WORKDIR /app` directive placed above this in the file, then our `bin/bash` would have failed.

Reply:
Right. In changing the image to be `FROM base` instead of `FROM nginx` I inherited the `WORKDIR`. Which was useful for running `npm run build` but required the change to the bash path. Glad I'm not going crazy after all.